hooks/usePlayerActions/createEmptyGameStreetActions.ts

```diff
@@ -0,0 +1,20 @@
+import { GameStreetActionType } from "../../models/GameSituation";
+import { Street } from "./usePlayerActions";
+
+export default function createEmptyGameStreetActions(
+  street: Street,
+  playerLength?: number
+) {
```

Comment:
I think it would be better to receive arguments as named parameters because the function name doesn't indicate what params and in what order to take them.

```suggestion
export default function createEmptyGameStreetActions({ street, playerLength }: {
  street: Street,
  playerLength?: number
}) {
```